.github/workflows/clang-format-check.yml

```diff
@@ -0,0 +1,13 @@
+name: Clang-Format Check
+on: [push, pull_request]
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run clang-format style check for C/C++/Protobuf programs.
+      uses: jidicula/clang-format-action@v4.5.0
+      with:
+        clang-format-version: '13'
```

Comment:
Why something else than what we are using in `kokkos/kokkos`?

Reply:
because our kokkos clang-format check runs on jenkins?

Reply:
I copied the one from desul over here.

Reply:
My concern is rather that people have to use different `clang-format` versions to contribute to `Kokkos` repositories.

Reply:
Last I checked, we use `clang-format 8.0` in `kokkos/kokkos`. 🙂

Reply:
I think it's less painful to change this to clang-format-8 since we are only starting to enforce indentation here.